MAINTAINERS.md

```diff
@@ -25,29 +99,57 @@ git push -u origin release-$now
 scripts/have-news HEAD^ > have-news.md
 ```
 
-Create a release PR, pasting `have-news.md` into the body.  Then build the SDK:
+- [ ] Create the release PR.
+
+The above should have pushed state to GitHub to let you create a
+new PR to merge the release branch back to the main branch. The PR
+name should follow the convention of previous releases, which is
+currently `chore(release): publish _release_label_`.  Paste
+`have-news.md` as the description of the PR.  Follow the example
```

Comment:
Note that some releases (pismoA, for example) have had such a large `have-news.md` that it exceeded GitHub's limits (either the size of a single Issue/PR comment, or the size of a commit message when the PR merge process wanted to use the first PR comment contents as the merge commit comment body). Fairly small/frequent releases wouldn't suffer from this problem, but it might be worth a warning here.